hack/prometheus-rules.yaml

```diff
@@ -54,7 +54,7 @@ spec:
         ) > 85
       for: 5m
       labels:
-        severity: alert
+        severity: critical
```

Comment:
I suggest to flip the severity `ElasticsearchNodeDiskWatermarkReached == 85%` to `warning` and `ElasticsearchNodeDiskWatermarkReached == 90%` to `critical` here.

Reply:
Thanks! I've made the suggested changes.